package.json

```diff
@@ -47,6 +47,6 @@
   "private": false,
   "publishConfig": {
     "registry": "https://registry.npmjs.org/",
-    "tag": "next"
```

Comment:
@jpvajda I think this will publish the release to npm under `latest` instead of under `next`... we might need to override this value in some way in the GH action so it always tags this branch with `next` when building the release (regardless of what the package.json file says)

Reply:
I changed the release tag to "next", will this not suffice?

Reply:
👍🏼